sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/AppView.groovy

```diff
@@ -84,6 +98,8 @@ class AppView extends VerticalLayout {
 
     private static ToggleButton setupProjectSampleToggle() {
         ToggleButton toggleButton = new ToggleButton("Show Samples", "Show Projects")
+        toggleButton.setEnabled(false)
```

Comment:
I think it might be a good idea to set it enabled for a given predicate/ check here the condition. This works as well though

Reply:
My idea was that when adding the button to the view nothing is loaded, so by selecting a project enabling will be triggerd :)